packages/lit-ssr/src/lib/render-lit-html.ts

```diff
@@ -514,7 +514,8 @@ export function* renderTemplateResult(
           // eslint-disable-next-line @typescript-eslint/no-explicit-any
           (null as any) as HTMLElement,
           op.name,
-          statics
+          statics,
+          undefined
```

Comment:
If this argument is optional, can it be removed?

Reply:
Done